packages/nodebox/package.json

```diff
@@ -3,7 +3,15 @@
   "version": "0.0.39",
   "description": "Public API of Nodebox",
   "main": "./build/index.js",
+  "module": "./build/index.mjs",
```

Comment:
This is only for `bundlers`. Some bundlers still use this field instead of exports. Or else, it's not a standard `package.json` field